### PR TITLE
Add digital diamonds ingo

### PR DESCRIPTION
**Pull request by gopatrik**

### CODE REVIEW

File: data.json [hunk 1]
```diff
@@ -2,7 +2,10 @@
   "teams": [
     {
       "name": "Digital Diamonds",
-      "members": [
+      "description": "Queue songs to any users Spotify Playlist via a tweet by @-ing the user, linking the song and adding #queuesong!",
+      "image": "/images/teams/digitaldiamonds.jpg",
+      "repo": "https://github.com/diversify/digitaldiamonds",
+      "members": ,
```
Comment (johanbrook) [inline data.json:8]: `members` should be an array :)

File: data.json [hunk 2]
```diff
@@ -2,7 +2,10 @@
   "teams": [
     {
       "name": "Digital Diamonds",
-      "members": [
+      "description": "Queue songs to any users Spotify Playlist via a tweet by @-ing the user, linking the song and adding #queuesong!",
+      "image": "/images/teams/digitaldiamonds.jpg",
```
Comment (johanbrook) [inline data.json:6]: Did you forget to add the image? :)

File: data.json (hunk 1 above)
Comment (gopatrik) [inline data.json:8]: How clumpsy of me! The image is added to my fork; don't know why it isn't added to the pull req! :/ Will try to check it out later, studying for an exam I have tomorrow!

**Comment by EjayJanis** [comment]
How can I send a gravatar of me on git? I don't have any pic on my profile and it's so lame. =)

**Comment by johanbrook** [comment]
@EjayJanis As I wrote in the [issue](https://github.com/diversify/diversify.github.io/issues/1), you can go to [Gravatar.com](gravatar.com) and sign up for an account, and add an image for the email address specified for you in `data.json` :) 

That image will then be used by other sites who support Gravatar! :)

**Comment by EjayJanis** [comment]
Awesome! I just thought that I had to send it to someone besides making an account in Gravatar =) Thank you for helping! The site is Fabulous!

**Comment by johanbrook** [comment]
Great! Thanks :)

**Pull request closed without merging**